llvm/test/DebugInfo/print-non-instruction-debug-info.ll

```diff
@@ -0,0 +1,80 @@
+;; Test that we can write in the new debug info format.
```

Comment:
There is a whitespace check in the test as written, the `{{^}}` at the start basically causes us to match leading whitespace. `--strict-whitespace` won't match leading whitespace unless you also pass `--match-full-lines`, and it's easier if we don't pass that since we don't have to match on the contents of the instructions.